jekyll/_cci2/insights.md

```diff
@@ -6,130 +6,88 @@ description: "Viewing the status of repos and test performance"
 order: 41
 version:
 - Server v2.x
+- Cloud
 ---
 
-<div class="alert alert-warning" role="alert">
-  <p><span style="font-size: 115%; font-weight: bold;">⚠️ Heads up!</span></p>
-  <span> This document refers to using the Insights page on the CircleCI <i>Server</i> product. If you are interested in accessing insights and analytics for your usage, please consider exploring the <a href="https://circleci.com/docs/api/v2/#tag/Insights">Insights endpoints</a> of the CircleCI V2 API.</span>
-</div>
-
-
 ## Overview
 
-Click the Insights menu item in the CircleCI app to view a dashboard showing the health of all repositories you are following. Median build time, median queue time, last build time, success rate, and parallelism appear for your default branch. **Note:** If you have configured Workflows, graphs display all of the jobs that are being executed for your default branch.
-
-![header]({{ site.baseurl }}/assets/img/docs/insights-1.0.gif)
-
-The image illustrates the following data about your builds:
-
-- Status of all your repos building on CircleCI in real time
-
-- Median queue time
-
-- Median build time
 
-- Number of branches
+The CircleCI Insights feature provides a dashboard for viewing the health and
+usage of your repositories build processes. _Insights_ provides time-series data
+overviews of credit usage, success rates, pipeline duration, and other pertinent
+information.
 
-- Last build
+This document describes how to access the Insights feature on CircleCI Cloud and Server.
```

Comment:
Insights 1.0 is now (mostly) only on Server, while Insights 2.0 is only on Cloud